docs/source/usage/cli.rst

```diff
@@ -170,11 +170,13 @@ Image
 
 Image-based provisioning can be accomplished through the use of `Metalsmith`_. It requires the image to be uploaded into OpenStack Glance. Once that's done, a non-admin can run the following:
 
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-|                | **Actions**                                                                                                         |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>`` |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
++----------------+--------------------------------------------------------------------------------------------------------------------+
+|                | **Actions**                                                                                                        |
++----------------+--------------------------------------------------------------------------------------------------------------------+
+| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>``|
++----------------+---------------------------------------- ---------------------------------------------------------------------------+
```

Comment:
I think the table is not rendered correctly.

Reply:
Oh whoops, Danni is right - you need a space before the last '|'. Can you just make sure this is fixed in the other metalsmith candidate PR?

Reply:
Sure.